datasets/objectenopenbareruimte/dataset.json

```diff
@@ -605,6 +605,262 @@
+              "unit": "m"
+            },
+            "maaifrequentie": {
+              "auth": "FP/MDW",
+              "description": "Aantal keren dat een groenobject of een terreindeel gemaaid wordt per jaar.",
+              "type": "string"
+            },
+            "mutatiedatum": {
+              "auth": "FP/MDW",
+              "description": "Datum waarop de digitale gegevens van het beheerobject voor het laatst gewijzigd zijn.",
+              "type": "string",
+              "format": "date"
+            },
+            "objecttype": {
+              "description": "Kleinste functionele eenheid (beheerobject of hulpobject).",
+              "type": "string"
+            },
+            "objectBegintijd": {
+              "auth": "FP/MDW",
+              "description": "Creatiedatum (tijdstip) van het digitale beheerobject binnen het beheersysteem of in de bovenliggende georegistratie.",
+              "type": "string",
+              "format": "date-time"
+            },
+            "objectEindtijd": {
+              "auth": "FP/MDW",
+              "description": "Vervaldatum (tijdstip) van het digitale beheerobject binnen het beheersysteem of in de bovenliggende georegistratie.",
+              "type": "string",
+              "format": "date-time"            
+            },
+            "onderhoudsplichtige": {
+              "auth": "FP/MDW",
+              "description": "Organisatie die verantwoordelijk is voor het onderhoud van het beheerobject.",
+              "type": "string"
+            },
+            "opTalud": {
+              "auth": "FP/MDW",
+              "description": "Aanduiding of het beheerobject wel of niet op een hellend vlak ligt. De taludsteilte wordt apart geregistreerd. Overname uit BGT-object.",
+              "type": "boolean"
+            },
+            "opleverdatum": {
+              "auth": "FP/MDW",
+              "description": "Datum waarop het beheerobject daadwerkelijk opgeleverd is. Begin van de garantieperiode.",
+              "type": "string",
+              "format": "date"
+            },
+            "oppervlakteGis": {
+              "description": "Oppervlakte van het  beheerobject, overname van BGT/IMGeo.",
+              "type": "number",
+              "unit": "m2"
+            },
+            "percentageLoofbos": {
+              "auth": "FP/MDW",
+              "description": "Percentage loofbos (t.o.v. naaldbos) binnen een gemengd bos.",
+              "type": "number",
+              "multipleOf": 0.1,
+              "minimum": 0,
+              "maximum": 100
+            },
+            "risicotoeslag": {
+              "auth": "FP/MDW",
+              "description": "beschrijving volgt",
+              "type": "string"
+            },
+            "theoretischEindjaar": {
+              "auth": "FP/MDW",
+              "description": "Jaar dat het beheerobject aan het theoretische einde van haar levensduur is.",
+              "type": "number"
```

Comment:
Is hier bewust voor `number` gekozen als type? Bij een jaar zou ik een `integer` verwachten.